src/components/SvgButton/SvgButton.js

```diff
@@ -12,6 +12,7 @@ const baseStyles = ({ theme }) => css`
   border: none;
   cursor: pointer;
   fill: ${theme.colors.black};
+  overflow: initial;
```

Comment:
Does this fix the bug in Chrome with Storybook?

Reply:
Yea. Weird, right :)